### PR TITLE
bnc#893983 - yast2-auth-client has hard requires for sssd

### PR DESCRIPTION
**Pull request by varkoly**

### CODE REVIEW

**Comment by jreidinger** [comment]
travis failed, please check why it cannot import namespace, maybe some syntax error?

File: package/yast2-auth-client.changes [hunk 1]
```diff
@@ -1,4 +1,15 @@
 -------------------------------------------------------------------
+Mon Jan  5 17:51:59 UTC 2015 - varkoly@suse.com
+
+- bnc#893983 - yast2-auth-client has hard requires for sssd
```
Comment (mvidner) [inline package/yast2-auth-client.changes]: The message does not make it clear whether the hard requires is the buggy state or the fixed state. IMHO better: "Removed an unnecessary hard dependency on sssd."

File: src/modules/AuthClient.rb [hunk 2]
```diff
@@ -121,6 +121,8 @@ def Write
       filter_groups = []
       filter_users  = []
       to_install    = []
+      if !Package.Installed("sssd") && Package.Available("sssd")
+          to_install << "sssd"
```
Comment (mvidner) [inline src/modules/AuthClient.rb]:
```ruby
end
```

**Comment by jreidinger** [comment]
LGTM

**Pull request merged**